tests/unit/utils/test_thin.py

```diff
@@ -30,6 +30,13 @@
 except ImportError:
     pytest = None
 
+if not salt.utils.thin.has_immutables:
+
+    class immutables:
+        __file__ = ""
+
+    salt.utils.thin.immutables = immutables
```

Comment:
Don't patch directly, use `mock.patch`.
This can bite us back in the future, as it has in the past...